bluemira/geometry/coordinates.py

```diff
@@ -1779,6 +1779,17 @@ def __iter__(self):
         """
         return iter(self._array)
 
+    def copy(self):
+        """
+        Copy of itself
+
+        Returns
+        -------
+        An instance of Coordinates, with exactly the same class and coordinates,
+        but sharing a different underlying copy of the data array.
+        """
+        return self.__class__(self.as_dict())
```

Comment:
This is a bit dangerous because the underlying arrays are not copied therefore the whole stack of copied coords is mutable. Also 
```suggestion
        return type(self)(self.as_dict())
```